packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -218,14 +264,13 @@ const ProofOfOwnershipForm = ({
                         <Button
                             type='submit'
                             className={classNames('account-form__footer-btn')}
-                            is_disabled={is_submit_button_disabled}
+                            is_disabled={!dirty || (dirty && !isValid)}
```

Comment:
```suggestion
                            is_disabled={!dirty || !isValid}
```
@jim-deriv nit-pick)